### ISSUE TITLE
escaping '&' in the items or 'Exception ('itemtype', '')'

### ISSUE THREAD

**Issue by gour** [dagraham/etm-dgraham#3]
Hello,

while migrating my taskwarrior tasks to etm I stumbled upon one which looks as: 
`basic & extended insurance` which contains special character `&`. I tried to escape it by using `basic \& extended`, but that does not work - `&` is not escaped, so wonder if there is some possibility to escape `&` and use it in the items?

Below is what I got:

```
$  (etm) etm

Unhandled exception in event loop:
  File "/usr/lib64/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/application/application.py", line 653, in read_from_input
    self.key_processor.process_keys()
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/key_binding/key_processor.py", line 274, in process_keys
    self._process_coroutine.send(key_press)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/key_binding/key_processor.py", line 186, in _process
    self._call_handler(matches[-1], key_sequence=buffer[:])
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/key_binding/key_processor.py", line 329, in _call_handler
    handler.call(event)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/key_binding/key_bindings.py", line 101, in call
    self.handler(event)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/key_binding/bindings/named_commands.py", line 270, in self_insert
    event.current_buffer.insert_text(event.data * event.arg)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/buffer.py", line 1209, in insert_text
    self.document = Document(text, cpos)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/buffer.py", line 536, in document
    self.set_document(value)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/buffer.py", line 571, in set_document
    self._cursor_position_changed()
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/buffer.py", line 516, in _cursor_position_changed
    self.on_cursor_position_changed.fire()
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/utils.py", line 77, in fire
    self()
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/prompt_toolkit/utils.py", line 73, in __call__
    handler(self.sender)
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/etm/view.py", line 1983, in default_cursor_position_changed
    set_askreply('_')
  File "/home/gour/.virtualenvs/etm/lib/python3.8/site-packages/etm/view.py", line 2723, in set_askreply
    ask, reply = item.askreply[('itemtype', '')]

Exception ('itemtype', '')
Press ENTER to continue...                               
```

Sincerely,
Gour

**Comment by dagraham** [comment]
The problem isn't the "&" but instead that the reminder does not begin with an item type character followed by a space. These are the item type characters:

`-` (task)
`*` (event)
`%` (record)
`!` (inbox)

If you try, e.g.,  
    `%  basic & extended insurance`
I think you will find that it works fine.

**Comment by gour** [comment]
> The problem isn't the "&" but instead that the reminder does not begin with an item type character followed by a space. 

No, I just mentioned the problem and not referencing the whole item which was in this case task type `-`.

> If you try, e.g.,
> `% basic & extended insurance`
> I think you will find that it works fine.

Created a new pristine venv and tried with:

```
- basic &
```
and as soon as I type `&` I get the same crash. Plese, try to reproduce?

**Comment by dagraham** [comment]
Fixed in 4.5.2.

**Comment by gour** [comment]
> Fixed in 4.5.2

Thanks a lot - it can be closed now.
